cmd/server.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"github.com/rs/zerolog/log"
+	"github.com/sevlyar/go-daemon"
+	"github.com/spf13/cobra"
+	command "github.com/threefoldtech/tf-grid-cli/pkg/cmd"
+)
+
+var serverCmd = &cobra.Command{
+	Use:   "server",
+	Short: "Run an rpc server listening for incoming commands to the tfgrid client",
```

Comment:
There is a language mistake here: "Run a rpc server..."

Reply:
i think this depends on how you pronounce RPC, if you pronounce it as "remote procedure call" then it should indeed be "a rpc", but if you pronounce it as "RPC", then i think it should be "an RPC", since R is pronounce "ARR" :D